src/tests/the_if_function.test.js

```diff
@@ -0,0 +1,5 @@
+const _if = require("../challenges/the_if_function.js")
+
+test("true should call func1", () => {
+  expect(_if(true, func1, func2)).toBe(func1())
```

Comment:
It looks like `_if` expects a boolean and 2 functions as inputs.  `true` is the boolean, but what is func1? and func2? They are undefined.